### ISSUE TITLE
Can I create pull requests?

### ISSUE THREAD

**Issue by 9fo** [ErinTomorri/automatic-tiktok-reddit-content-creator#1]
I am very advanced in TikTok API, if u would like I can make a pull request to fix ur uploader with requests?

**Comment by ErinTomorri** [comment]
Yeah go ahead that would be a great help

**Comment by 9fo** [comment]
> Yeah go ahead that would be a great help

Does ur current upload.py work? Because I see u have updated it.

**Comment by ErinTomorri** [comment]
No, I never got it to work, so I deleted it. I try to create a new uploader later today.

**Comment by ErinTomorri** [comment]
erin #3443 on discord

**Comment by 9fo** [comment]
> erin #3443 on discord

Added

**Comment by romanu77** [comment]
it's working now?

**Comment by ErinTomorri** [comment]
The code itself works (as in the video gets made) but you still need to upload the video manually

**Comment by ErinTomorri** [comment]
sure thing add me on discord

**Comment by ErinTomorri** [comment]
https://discord.gg/5AVV5dSF3z if anyone is experiencing issues, join this discord server to ask questions or add me on discord

**Comment by ErinTomorri** [comment]
Post your code that you have right now and I can try to fix it for you. I got mine to work after playing around with it.
